collect_app/src/main/java/org/odk/collect/android/widgets/AbstractSelectOneWidget.java

```diff
@@ -90,15 +96,19 @@ public IAnswerData getAnswer() {
                     : new SelectOneData(new Selection(selectChoice));
     }
 
-    protected void createLayout() {
+    protected void createLayout(Context context) {
         adapter = new SelectOneListAdapter(items, selectedValue, this, numColumns, this.getFormEntryPrompt(), this.getReferenceManager(), this.getAnswerFontSize(), this.getAudioHelper(), getPlayColor(getFormEntryPrompt(), themeUtils), this.getContext(), autoAdvance);
 
         if (items != null) {
             RecyclerView recyclerView = setUpRecyclerView();
             recyclerView.setAdapter(adapter);
             answerLayout.addView(recyclerView);
             adjustRecyclerViewSize(adapter, recyclerView);
-            addAnswerView(answerLayout);
+
+            Resources resources = context.getResources();
+            int marginStandard = dpFromPx(context, resources.getDimensionPixelSize(R.dimen.margin_standard));
+            int margin = marginStandard - FIELD_HORIZONTAL_MARGIN_MODIFIER;
+            addAnswerView(answerLayout, margin);
```

Comment:
We don't want to any of the select widgets as we need the select options to spread from edge to edge. This change should be reverted.